### PR TITLE
v8.4: update CentOS 7-related descriptions

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Update CentOS 7-related descriptions for TiDB v8.4.0.

Note: "CentOS 7" also exists in many other documents, which will be updated as needed later in another PR.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: backup-and-restore-using-dumpling-lightning.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ summary: 了解如何使用 Dumpling 和 TiDB Lightning 备份与恢复集群数
 
 ## 资源要求
 
-**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 <!--TODO--> 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
```
Comment (Oreoxmt) [inline backup-and-restore-using-dumpling-lightning.md:33]: @Benjamin2037 PTAL

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
 
     <div label="Linux">
 
-    对于 Linux 操作系统，下面以 CentOS 7 为例：
+    对于 Linux 操作系统，下面以 CentOS 7 为例：<!--TODO-->
```
Comment (Oreoxmt) [inline develop/dev-guide-build-cluster-in-cloud.md:90]: @Icemap PTAL

File: dm/dm-hardware-and-software-requirements.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ TiDB Data Migration (DM) 支持主流的 Linux 操作系统，具体版本要求
 | Linux 操作系统       | 版本         |
 | :----------------------- | :----------: |
 | Red Hat Enterprise Linux | 7.3 及以上   |
-| CentOS                   | 7.3 及以上   |
+| CentOS                   | 7.3 及以上  <!--TODO--> |
```
Comment (Oreoxmt) [inline dm/dm-hardware-and-software-requirements.md:14]: @Benjamin2037 PTAL

File: ticdc/deploy-ticdc.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ summary: 了解 TiCDC 软硬件环境要求以及如何安装部署和运维 TiC
 | Linux 操作系统       | 版本         |
 | :----------------------- | :----------: |
 | Red Hat Enterprise Linux | 7.3 及以上   |
-| CentOS                   | 7.3 及以上   |
+| CentOS                   | 7.3 及以上  <!--TODO--> |
```
Comment (Oreoxmt) [inline ticdc/deploy-ticdc.md:17]: @flowbehappy PTAL

File: tidb-lightning/tidb-lightning-logical-import-mode.md [hunk 5]
```diff
@@ -13,7 +13,7 @@ summary: 了解 TiDB Lightning 的逻辑导入模式 (Logical Import Mode)。
 
 **操作系统**：
 
-建议使用新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
+建议使用新的、纯净版 CentOS 7 <!--TODO--> 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-logical-import-mode.md:16]: @Benjamin2037 PTAL

File: tidb-lightning/tidb-lightning-physical-import-mode.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ backend = "local"
 
 ### 运行环境需求
 
-**操作系统**：建议使用新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
+**操作系统**：建议使用新的、纯净版 CentOS 7 <!--TODO--> 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
```
Comment (Oreoxmt) [inline tidb-lightning/tidb-lightning-physical-import-mode.md:52]: @Benjamin2037 PTAL

File: tiup/tiup-component-cluster-check.md [hunk 7]
```diff
@@ -11,7 +11,7 @@ summary: TiUP Cluster 提供了 `check` 子命令，用于检查集群的硬件
 
 ### 操作系统版本
 
-检查部署机操作系统发行版和版本：目前仅支持部署在 CentOS 7 的操作系统上，之后随兼容性改进可能支持更多系统版本。
+检查部署机操作系统发行版和版本：目前仅支持部署在 CentOS 7 的操作系统上，之后随兼容性改进可能支持更多系统版本。<!--TODO-->
```
Comment (Oreoxmt) [inline tiup/tiup-component-cluster-check.md:14]: @bb7133 PTAL

File: tune-operating-system.md [hunk 8]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs-cn/dev/tune-operating-system/']
 
 # 操作系统性能参数调优
 
-本文档仅用于描述如何优化 CentOS 7 的各个子系统。
+本文档仅用于描述如何优化 CentOS 7 的各个子系统。<!--TODO: review this document and confirm whether it's suitable for TiDB v8.4-->
```
Comment (Oreoxmt) [inline tune-operating-system.md:9]: @bb7133 PTAL

File: migrate-large-mysql-to-tidb.md [hunk 9]
```diff
@@ -19,7 +19,7 @@ summary: 介绍如何从大数据量 MySQL 迁移数据到 TiDB。
 
 ## 资源要求
 
-**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
+**操作系统**：本文档示例使用的是若干新的、纯净版 CentOS 7 <!--TODO--> 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。
```
Comment (Oreoxmt) [inline migrate-large-mysql-to-tidb.md:22]: @Benjamin2037 PTAL

File: hardware-and-software-requirements.md [hunk 10]
```diff
@@ -21,7 +21,7 @@ TiDB 作为一款开源一栈式实时 HTAP 数据库，可以很好地部署和
 |  操作系统   |   支持的 CPU 架构   |
 |   :---   |   :---   |
 | Red Hat Enterprise Linux 8.4 及以上的 8.x 版本  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
-| <ul><li>Red Hat Enterprise Linux 7.3 及以上的 7.x 版本</li><li>CentOS 7.3 及以上的 7.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>   |
+| <ul><li>Red Hat Enterprise Linux 7.3 及以上的 7.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>   |
```
Comment (Oreoxmt) [inline hardware-and-software-requirements.md:24]: @bb7133 PTAL

File: best-practices/haproxy-best-practices.md [hunk 11]
```diff
@@ -53,7 +53,7 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
 | Linux 操作系统       | 版本         |
 | :----------------------- | :----------- |
 | Red Hat Enterprise Linux | 7 或者 8   |
-| CentOS                   | 7 或者 8   |
+| CentOS                   | 7 或者 8 <!--TODO--> |
```
Comment (Oreoxmt) [inline best-practices/haproxy-best-practices.md:56]: @bb7133 PTAL

File: check-before-deployment.md
Comment (Oreoxmt) [inline check-before-deployment.md:1]: @bb7133 PTAL

File: quick-start-with-tidb.md
Comment (Oreoxmt) [inline quick-start-with-tidb.md:1]: @bb7133 PTAL

File: develop/dev-guide-build-cluster-in-cloud.md (hunk 2 above)
Comment (Icemap) [reply develop/dev-guide-build-cluster-in-cloud.md]: Done

File: ticdc/deploy-ticdc.md (hunk 4 above)
Comment (wlwilliamx) [reply ticdc/deploy-ticdc.md:17]: Done

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 12]
```diff
@@ -87,10 +87,10 @@ aliases: ['/zh/tidb/dev/build-cluster-in-cloud']
 
     <div label="Linux">
 
-    对于 Linux 操作系统，下面以 CentOS 7 为例：
+    对于 Linux 操作系统，下面以 Ubuntu 为例：
```
Comment (jebter) [inline develop/dev-guide-build-cluster-in-cloud.md:90]: 为什么是 ubuntu 而不是 rocky9？

Comment (jebter) [reply develop/dev-guide-build-cluster-in-cloud.md:90]: 这是 client 端

**Review by stevenayu — APPROVED** [review]

File: hardware-and-software-requirements.md [hunk 13]
```diff
@@ -40,7 +40,7 @@ TiDB 作为一款开源一栈式实时 HTAP 数据库，可以很好地部署和
 > **注意：**
 >
 > - TiDB 只支持 Red Hat 兼容内核 (RHCK) 的 Oracle Enterprise Linux，不支持 Oracle Enterprise Linux 提供的 Unbreakable Enterprise Kernel。
-> - 根据 [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/)，CentOS Linux 8 的上游支持已于 2021 年 12 月 31 日终止，但 CentOS 将继续提供对 CentOS Stream 8 的支持。
+> - 根据 [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/)，CentOS Linux 7 的上游支持于 2024 年 6 月 30 日终止。TiDB 从 v8.4.0 版本开始结束对 CentOS 7 的支持，建议使用 Rocky Linux 9.1 及以上的版本。CentOS Linux 8 的上游支持已于 2021 年 12 月 31 日终止，但 CentOS 将继续提供对 CentOS Stream 8 的支持。
```
Comment (qiancai) [inline hardware-and-software-requirements.md:43]:
```suggestion
> - 根据 [CentOS Linux EOL](https://www.centos.org/centos-linux-eol/)，CentOS Linux 7 的上游支持于 2024 年 6 月 30 日终止。从 v8.4.0 版本开始，TiDB 已结束对 CentOS 7 的支持，建议使用 Rocky Linux 9.1 及以上的版本。CentOS Linux 8 的上游支持已于 2021 年 12 月 31 日终止，但 CentOS 将继续提供对 CentOS Stream 8 的支持。
```

File: ticdc/deploy-ticdc.md [hunk 14]
```diff
@@ -9,18 +9,13 @@ summary: 了解 TiCDC 软硬件环境要求以及如何安装部署和运维 TiC
 
 ## 软件和硬件环境推荐配置
 
-在生产环境中，TiCDC 的软件和硬件配置推荐如下：
-
-| Linux 操作系统       | 版本         |
-| :----------------------- | :----------: |
-| Red Hat Enterprise Linux | 7.3 及以上   |
-| CentOS                   | 7.3 及以上   |
+在生产环境中，TiCDC 硬件配置推荐如下：
```
Comment (qiancai) [inline ticdc/deploy-ticdc.md:12]: L14～L16 要留保留吗

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**